src/Utility/Text.php

```diff
@@ -564,124 +565,241 @@ public static function tail($text, $length = 100, array $options = [])
     public static function truncate($text, $length = 100, array $options = [])
     {
         $default = [
-            'ellipsis' => '...', 'exact' => true, 'html' => false
+            'ellipsis' => '...', 'exact' => true, 'html' => false, 'trimWidth' => false,
```

Comment:
It would be worth to add `Text::truncateWidth()`, because Japanese or other East Asian users would need to set `trimWidth` to `true` always. I'll add it tomorrow.